openapi/components/schemas/CustomField.yaml

```
@@ -40,9 +40,14 @@ properties:
 
       Additional parameters which can be added according to type:
 
-      | Parameter name | Types         | Description |
-      | -------------- | ------------- | ----------- |
-      | `allowedValues`| string, array | List of allowed values. |
-      | `maxLength`    | string        | Maximum allowed length of a string. A maximum length of 4000 characters can be configured. By default, this value is set to 255.
+      | Parameter name | Types            | Description |
+      | -------------- | ---------------- | ----------- |
+      | `allowedValues`| string, array    | List of allowed values. |
+      | `maxLength`    | string           | Maximum allowed length of a string. A maximum length of 4000 characters can be configured. By default, this value is set to 255. |
+      | `minimum`      | number, integer  | Minimum allowed value of a number or integer. It combines with `maximum`. |
+      | `maximum`      | number, integer  | Maximum allowed value of a number or integer. It combines with `minimum`. |
+      | `pattern`      | string           | Regular expression allowed of a string. |
```

suggestion: to use a similar structure to the other descriptions
(I found this one difficult to understand)

```suggestion
      | `pattern`      | string           | Allowed regular expression of a string. |
```